staging/src/k8s.io/pod-security-admission/admission/admission.go

```diff
@@ -379,6 +379,7 @@ func (a *Admission) EvaluatePod(ctx context.Context, namespaceName string, podMe
 		klog.V(2).InfoS("failed to parse PodSecurity namespace labels", "err", err)
 		auditAnnotations["error"] = fmt.Sprintf("Failed to parse policy: %v", err)
 	}
+	klog.V(4).InfoS("Evaluating", "resource", attrs.GetResource(), "name", attrs.GetName(), "namespace", attrs.GetNamespace(), "policy", nsPolicy)
```

Comment:
this will log "evaluating ... resource=...namespaces" when updating a namespace, which seems weird